webgpu.h

```diff
@@ -800,6 +800,9 @@ typedef void (*WGPUProcComputePassEncoderPushDebugGroup)(WGPUComputePassEncoder
 typedef void (*WGPUProcComputePassEncoderSetBindGroup)(WGPUComputePassEncoder computePassEncoder, uint32_t groupIndex, WGPUBindGroup group, uint32_t dynamicOffsetCount, uint32_t const * dynamicOffsets);
 typedef void (*WGPUProcComputePassEncoderSetPipeline)(WGPUComputePassEncoder computePassEncoder, WGPUComputePipeline pipeline);
 
+// Procs of ComputePipeline
+typedef WGPUBindGroupLayout (*WGPUProcComputePipelineGetBindGroupLayout)(WGPUComputePipeline computePipeline, uint32_t group);
```

Comment:
the working group hasn't agreed on that ability in https://github.com/gpuweb/gpuweb/issues/446

Reply:
Looking at the [minutes](https://docs.google.com/document/d/18CnUBiiUKW11zGsM_RuAkaIBv-O4s3txFuGqcQIb6sw/) what I get from them is that it was agreed upon with just a discussion left on whether the same JS objects are returned or not.

Reply:
There is not even a PR open to the spec! We shouldn't rush in here before the upstream is done.

I don't think the discussion is done either, e.g. was this addressed?
> I do wonder though, how would this play with WSL shaders? Overall, since we have to write down in the specification the deriving rules from the shading language, this PR might be blocked until we settle on what that language is.

Reply:
> There is not even a PR open to the spec! We shouldn't rush in here before the upstream is done.

I guess we had a different understanding of how accepted this was. I'll temporarily remove it from what's generated for this webgpu.h.

> MM: sounds fine. Believe it is possible to create a pipeline layout from WSL shader.

That's from the minutes, but I guess it is a "likely ok but need to check"?

Reply:
Man, I wish people documented their decisions in the appropriate discussions instead of waiting till the last minute to voice them on a call/f2f :/